src/test/resources/data/AllAccessControlUiTest.json

```diff
@@ -1069,7 +1069,7 @@
       "courseId": "idOfTypicalCourse1.access",
       "feedbackQuestionId": "2",
```

Comment:
It would be great if you could help fix the error in question id as well. (I believe it should be 1.)